### PR TITLE
Slight sizing error

### PR DESCRIPTION
**Pull request by charleslparker**
For some combinations of input size / image size, there was a bug causing the resized image to be off by one pixel, which was causing the concatenation errors in the Yolov4 predictions spotted by @unmonoqueteclea.  I'm kind of shocked that this hasn't surfaced until now, and it's possible that this has impacted predictions using the tinyyolo models as well, though it seems like the impact of an extra row or column in the input image should be small.

### CODE REVIEW

File: sensenet/layers/extract.py [hunk 1]
```diff
@@ -6,10 +6,12 @@ def add(config, layer):
...
-    return {
-        "type": "activation",
-        "activation_function": layer.activation.__name__,
-    }
+    if "negative_slope" in config and "max_value" in config:  # ReLU layer
+        afn = "relu"
+    else:
+        afn = layer.activation.__name__
+
+    return {"type": "activation", "activation_function": afn}
```
Comment (unmonoqueteclea) [inline sensenet/layers/extract.py:14]: Is this something that can help me with my never ending plates project?

**Review by unmonoqueteclea — APPROVED** [review]
Thank you very much, Charlie.

One question, just in case. Was this bug affecting Pyramid training process or just Sensenet scoring?

When I started training YoloV4 models I changed also the input image size. That's why I thought the problem was related with YoloV4 and not with the combination if input/image size.

**Comment by unmonoqueteclea** [comment]
@charleslparker As I think you will update Pyramid to use the new version of Sensenet, I would need you to include there (and in shapsplain) the  trick you added in the setup.py file to avoid installing tensorflow if tensorflow-gpu is installed. Could that be
possible? Thanks

File: sensenet/layers/extract.py (hunk 1 above)
Comment (charleslparker) [reply sensenet/layers/extract.py:14]: > Is this something that can help me with my never ending plates project?

Nope!  Unrelated.

**Comment by unmonoqueteclea** [comment]
@charleslparker , no problem with Pyramid. I just remembered that I maintain my own fork of Pyramid specifically for that project (adding a LICENSE file, and that trick in the setup.py) So, I think I just need it in shapsplain

**Pull request merged**